### DISCUSSION TITLE
mail.domain.xyz does not use encryption

### DISCUSSION THREAD

**Discussion by ukwuezeobinna** [modoboa/modoboa#2515] [Q&A]
I get the below warning when trying to connect with thunderbird. What are the required settings that will help me enable encryption for incoming and outgoing emails.

"Incoming settings:

mail.domain.xyz does not use encryption.

Insecure mail servers do not use encrypted connections to protect your passwords and private information. By connecting to this server you could expose your password and private information.
Outgoing settings:

mail.domain.xyz does not use encryption.

Insecure mail servers do not use encrypted connections to protect your passwords and private information. By connecting to this server you could expose your password and private information."

**Comment by tonioo** [comment]
What is your current configuration?

**Reply by ukwuezeobinna** [reply]
What is the recommended configuration?

I have played around with these settings for sometime, would it be possible to re-install the server without loosing the data?

![Screenshot 2022-05-18 at 10-17-39 Modoboa settings](https://user-images.githubusercontent.com/852830/169006450-181ac860-6625-4d34-a6ad-b4cb9ea26510.png)

![Screenshot 2022-05-18 at 10-17-02 Modoboa settings](https://user-images.githubusercontent.com/852830/169006568-4071493b-d453-44c2-ae04-6c75c8308b9e.png)

**Comment by ukwuezeobinna** [comment]
Not related but the new Modoboa V2 UI is amazing.

**Comment by tonioo** [comment]
Those settings are for the webmail only, they won't affect mail clients like Thunderbird. I suggest you use the following config:
- IMAP : port 143 / STARTTLS or port 993 / SSL
- SMTP : port 587 / STARTTLS
